tests/fig28-lutff-pair/README.md

```diff
@@ -0,0 +1,15 @@
+## PAIR tests
```

Comment:
This test is already at https://github.com/SymbiFlow/python-symbiflow-v2x/tree/master/tests/vtr/lutff-pair